packages/mds-transaction-api/handlers/get-transactions.ts

```diff
@@ -240,21 +257,29 @@ export const GetTransactionsAsCsvHandler = async (
     ]
 
     const providers = await ProviderServiceClient.getProviders({})
+
+    const providerMap = providers.reduce<Record<UUID, ProviderDomainModel>>((acc, provider) => {
+      acc[provider.provider_id] = provider
+
+      return acc
+    }, {})
+
     const options = {
-      provider_id,
+      provider_ids: provider_id ? [provider_id] : undefined,
```

Comment:
I guess these options, had they been passed through an Ajv validator would have caught the fact that there was an additional unexpected property.

Reply:
What layer would the AJV have been on?